packages/x-charts/src/BarChart/AnimatedBarElement.tsx

```diff
@@ -26,6 +26,7 @@ export function AnimatedBarElement(props: BarProps) {
   return (
     <animated.rect
       {...other}
+      // @ts-expect-error
       filter={ownerState.isHighlighted ? 'brightness(120%)' : undefined}
```

Comment:
Why does it error?

Reply:
Possibly, but AFAIK, we didn't go to the full extent of finding the root cause.